test/math/FloatBox2DTest.ooc

```diff
@@ -56,6 +76,73 @@ FloatBox2DTest: class extends Fixture {
 			shrunkBox := paddedBox shrink(1.0f / 11.0f)
 			expect(box == shrunkBox, is true)
 		})
+		this add("rounding", func {
+			_round := this box3 round()
+			_ceiling := this box3 ceiling()
+			_floor := this box3 floor()
```

Comment:
Hmm if I recall correctly I had some trouble somewhere were I was using a variable together with a function of the same name.. but here it seems to work fine. Rock works in mysterious ways. Will fix.